tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java

```diff
@@ -42,75 +53,98 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNull;
 
-public class DeleteOffsetsConsumerGroupCommandIntegrationTest extends ConsumerGroupCommandTest {
-    String[] getArgs(String group, String topic) {
-        return new String[] {
-            "--bootstrap-server", bootstrapServers(listenerName()),
-            "--delete-offsets",
-            "--group", group,
-            "--topic", topic
-        };
+@Tag("integration")
+@ClusterTestDefaults(clusterType = Type.ALL, brokers = 3, serverProperties = {
```

Comment:
Could we reduce the replica number instead of increasing number of brokers? I try to avoid our CI get burned :)